ipv-core/src/main/java/com/ibm/whc/deid/util/localization/LocalizationManager.java

```diff
@@ -130,7 +131,46 @@ private LocalizationManager() {
     }
   }
 
-  private synchronized boolean registerResource(Resource resource, ResourceEntry entry) {
+  /**
+   * Register an advanced custom resource

```

Comment:
I'd suggest changing this comment.

Reply:
Changing to "register a custom resource"